iree-requirements.txt

```diff
@@ -1,2 +1,2 @@
-iree-compiler
-iree-runtime
+iree-compiler==20240724.964
+iree-runtime==20240724.964
```

Comment:
These were pinned before https://github.com/iree-org/iree-turbine/pull/39, FYI

We might also want the `setup.py` changes from https://github.com/iree-org/iree-turbine/pull/21 🤔

Reply:
yeah, weird that we unpin haha

> we might also want setup.py changes

btw, did you mean `load_requirement_pins("iree-requirements.txt")` ?